test/test.html

```diff
@@ -1,3 +1,3 @@
 <!DOCTYPE html>
 <title>can-define tests</title>
-<script src="../node_modules/steal/steal.js" main="can-define/test/test"></script>
+<script src="../node_modules/steal/steal-with-promises.js" main="can-define/test/test"></script>
```

Comment:
Let's not change this here. The `canjs/canjs` repo is set up to use the `-with-promises.js` file for IE11. It's much easier to debug things with native promises, so we should just use `steal.js` in other projects.

Reply:
got it! will change to using just `steal.js` for the other projects, including this one.